pkg/agent/controller/traceflow/packetin.go

```diff
@@ -113,7 +123,7 @@ func (c *Controller) parsePacketIn(pktIn *ofctrl.PacketIn) (*crdv1alpha1.Tracefl
 	}
 	c.runningTraceflowsMutex.RUnlock()
 	if !exists {
-		return nil, nil, nil, fmt.Errorf("Traceflow for dataplane tag %d not found in cache", tag)
+		return nil, nil, nil, fmt.Errorf("traceflow for dataplane tag %d not found in cache", tag)
```

Comment:
This was actually correct. Error strings should not be capitalized **_unless_** beginning with proper nouns or acronyms.
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
ok, just change it back